velox/type/Type.cpp

```diff
@@ -658,6 +658,7 @@ KOSKI_DEFINE_SCALAR_ACCESSOR(BOOLEAN);
 KOSKI_DEFINE_SCALAR_ACCESSOR(TINYINT);
 KOSKI_DEFINE_SCALAR_ACCESSOR(SMALLINT);
 KOSKI_DEFINE_SCALAR_ACCESSOR(BIGINT);
+KOSKI_DEFINE_SCALAR_ACCESSOR(HUGEINT);
```

Comment:
`HUGEINT` is not a logical type, we should use the existing `LongDecimalType::create()` here